Maestro/subscriptions.json

```diff
@@ -1178,15 +1178,64 @@
         }
       }
     },
-    // Automate opening PRs to merge aspnet 3.0 branches back to master
+    // Automate opening PRs to merge aspnet 3.0 preview branches back to 3.0 stable branches
     {
       "triggerPaths": [
-        "https://github.com/aspnet/AspNetCore/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/EntityFramework6/blob/release/6.3-preview7/**/*",
-        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview7/**/*",   
-        "https://github.com/aspnet/Extensions/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/Blazor/blob/release/0.10.0-preview7/**/*",
+        "https://github.com/aspnet/AspNetCore/blob/release/3.0-preview8/**/*",
+        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0-preview8/**/*",
+        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview8/**/*",   
+        "https://github.com/aspnet/Extensions/blob/release/3.0-preview8/**/*",
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "release/3.0",
```

Comment:
@dagood any way to make this a variable like "trigger-branch less -preview8"? Would be nice to avoid the extra blocks for Blazor and EF6.

Reply:
Now only have an extra block for EF6 but still&hellip;

Reply:
Maestro doesn't do logic like this internally, but I could see making `github-dnceng-branch-merge-pr-generator` smart enough. If `vsoSourceBranch` is removed, the build def would be queued with the default branch. Then the script can be changed to do whatever's necessary with the params, like maybe add a `"HeadBranchWithDefaultBaseBranchBasedOnPrerelease": "<trigger-branch>"`.

Reply:
My comment was about my _intended_ change to `BaseBranch`, not what I did in this initial commit.

Reply:
I don't really understand what the difference is... anyway, what I'm saying is that this:

```json
      "actionArguments": {
        "vsoSourceBranch": "master",
        "vsoBuildParameters": {
          "GithubRepoOwner": "aspnet",
          "GithubRepoName": "<trigger-repo>",
          "HeadBranch": "<trigger-branch>",
          "BaseBranch": "release/3.0"
        }
```

Might be changeable to this with the appropriate Arcade-side changes:

```json
      "actionArguments": {
        "vsoBuildParameters": {
          "GithubRepoOwner": "aspnet",
          "GithubRepoName": "<trigger-repo>",
          "HeadBranchWithDefaultBaseBranchMadeByChoppingOffPrerelease": "<trigger-branch>"
        }
```

I'm not familiar enough with the planned changes to the subscriptions file to know if that's really worthwhile.

Reply:
Sounds nice 🥇, both for removing `vsoSourceBranch` and avoiding the minor repetition I needed in this PR.

Reply:
If you file I bug about either, please /cc me so I'm subscribed 😺

Reply:
I don't plan to, benefits are fairly minor IMO, and I'm hoping the process can move to Maestro++ entirely so this is managed in the same place as the other branch-specific stuff like subscriptions (and have more "bounded"/readable configuration, some error reporting, etc.): https://github.com/dotnet/arcade/issues/1141.